src/store/Network.ts

```diff
@@ -225,6 +231,18 @@ export default {
       if (oldGenerationHash != networkModel.generationHash) {
         dispatch('account/NETWORK_CHANGED', {}, { root: true })
         dispatch('statistics/LOAD', {}, { root: true })
+        console.log(oldGenerationHash, networkModel.generationHash, currentProfile.generationHash)
+        if (
```

Comment:
To get rid of the if block can we rephrase this block as => dispatch('SET_NETWORK_IS_NOT_MATCHING_PROFILE', currentProfile &&
          (currentProfile.networkType !== networkModel.networkType ||
            currentProfile.generationHash !== networkModel.generationHash))

Reply:
I think it's more readable with the if block.